src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,22 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => (
+  <ul>
+    {tabs.map(tab => (
+      <li
+        key={tab.id}
+        className={tab.id === activeTabId ? 'is-active' : ''}
+        data-cy="Tab"
+      >
+        <a
+          href={`#${tab.id}`}
+          data-cy="TabLink"
+          onClick={event => {
+            event.preventDefault();
+            onTabSelected(tab.id);
+          }}
```

Comment:
```suggestion
          onClick={event => {
            if (tab.id !== activeTabId) {
              onTabSelected(tab.id);
            }
          }}
```

to prevent event from execution if selected tab is clicked again (required in tests)